app/Http/Controllers/UserController.php

```diff
@@ -15,10 +15,14 @@
 use Illuminate\Auth\Events\Registered;
 
 class UserController extends Controller {
-    public function viewUserOverview(Request $request): View {
+    public function handleViewUserOverview(Request $request) {
         if ($request->user()->can('viewAny', User::class)) {
-            $users = User::getAllUserData();
-            return view('user.overview', ['users' => $users]);
+
+            $roles = Role::getAllRoleData();
+            $users = isset($request->column)
```

Comment:
als ik het goed begrijp kan je hiermee dus niet filteren en sorteren tegelijk? is dat iets wat je wel zou willen? hoeft van mij niet maar gwn een vraag

Reply:
Mij leek het fijner dat je met één klik een rij kan sorteren.